src/Microsoft.Health.Fhir.Api/Features/SMART/SmartClinicalScopesMiddleware.cs

```diff
@@ -32,10 +33,12 @@ public SmartClinicalScopesMiddleware(RequestDelegate next)
             _next = next;
         }
 
-        public async Task Invoke(HttpContext context, RequestContextAccessor<IFhirRequestContext> fhirRequestContextAccessor, AuthorizationConfiguration authorizationConfiguration)
+        public async Task Invoke(HttpContext context, RequestContextAccessor<IFhirRequestContext> fhirRequestContextAccessor, IOptions<SecurityConfiguration> securityConfigurationOptions)
         {
             EnsureArg.IsNotNull(fhirRequestContextAccessor, nameof(fhirRequestContextAccessor));
-            EnsureArg.IsNotNull(authorizationConfiguration, nameof(authorizationConfiguration));
+            EnsureArg.IsNotNull(securityConfigurationOptions, nameof(securityConfigurationOptions));
+
+            var authorizationConfiguration = securityConfigurationOptions.Value.Authorization;
 
             if (fhirRequestContextAccessor.RequestContext.Principal != null
                 && authorizationConfiguration.Enabled)
```

Comment:
Should this line be cleaned up or be changed to use `securityConfigurationOptions`?

Reply:
Can't seem to add comments in the parts of this file that wasn't changed.. but same goes for all other lines referencing `authorizationConfiguration`